packages/@aws-cdk/aws-codepipeline/lib/artifact.ts

```diff
@@ -51,6 +52,13 @@ export class Artifact {
   public toString() {
     return this.artifactName;
   }
+
+  public overrideAsset(asset: assets.Asset): { [name: string]: string } {
```

Comment:
Missing documentation. I am not sure about this name... Reads weird: `artifact.overrideAsset(asset)`. What does this mean? Maybe something like `artifact.wireAsset(asset)` or `artifact.useAsset(asset)`?